pkg/controller/account/ec2.go

```diff
@@ -284,11 +352,37 @@ func CreateEC2Instance(reqLogger logr.Logger, account *awsv1alpha1.Account, clie
 					Tags:         tags,
 				},
 			},
-		})
+		}
+		runResult, runErr := client.RunInstances(input)
 
 		// Return on unexpected errors:
 		if runErr != nil {
 			if aerr, ok := runErr.(awserr.Error); ok {
+				// Attempt to create new sameple VPC and specify subnet in input
+				reqLogger.Info(fmt.Sprintln("Failed to RunInstances. Try to create new VPC"))
+				subnetID, err := GetSubnetID(reqLogger, client)
```

Comment:
I think we should move this VPC creation up into the InitializeRegion function, so that it's clear where we're creating the VPC and destroying the VPC afterwards, where right now I got a bit confused thinking that the VPC didn't have any cleanup being run on it because I would have expected that to be within the same function.

Reply:
agreed. moved.